### PR TITLE
initial rhel8 ospp template

### PR DESCRIPTION
**Pull request by shawndwells**
Initial RHEL8 OSPP profile. Does not add new rules, rather adds verbose comments to show where rules should/should not go. Meant to orientate the conversation of what goes into OSPP.

Fully expected sections will add/remove/change as dialog continues.

### CODE REVIEW

File: rhel8/profiles/ospp.profile [hunk 1]
```diff
@@ -106,94 +304,532 @@ selections:
...
+
+
+
+
+    #######################################################
+    ## Rules that need a home?
+
+
+#    - sysctl_kernel_yama_ptrace_scope
+#    - sysctl_kernel_kptr_restrict
```
Comment (jan-cerny) [inline rhel8/profiles/ospp.profile]: Will we keep the rules that don't have a home in the profile? If there isn't appropriate category for them in OSPP, should they be removed?

Comment (shawndwells) [reply rhel8/profiles/ospp.profile]: Don't know. For now, commented out rules that I didn't know what to do with.... IIRC there are some "catch all" categories we can map rules into at our discretion.

**Comment by shawndwells** [comment]
rebased & ready for merge

**Comment by redhatrises** [comment]
lgtm.

**Pull request merged**